vagrant/README.md

```diff
@@ -145,7 +153,7 @@ NAME      mTLS
 default   on
 ```
 
-If you try to access the marketplace via http://192.168.33.20:18080, it won't work because that traffic goes through the dataplane and is now encrypted via mTLS.
+If you try to access the marketplace via http://localhost:8080, it won't work because that traffic goes through the dataplane and is now encrypted via mTLS.
```

Comment:
Access via `http://localhost:8080` will work because it goes directly into the app.

In contrast, access via `http://192.168.33.20:18080` won't work because it goes via dataplane.